akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingInternalsSpec.scala

```diff
@@ -0,0 +1,39 @@
+package akka.cluster.sharding
```

Comment:
Please add

```
/*
 * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
 */
```

as all files (before the package)

Reply:
done